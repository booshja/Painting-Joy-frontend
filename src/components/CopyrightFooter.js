// dependencies
import React from "react";
import styled from "styled-components";

const StyledP = styled.p`
    font-family: "Barlow Condensed", sans-serif;
    color: #ffffff;
    font-size: 1rem;
`;

const CopyrightFooter = () => {
    return <StyledP>Copyright &copy; 2021 - Painting Joy Mural Co.</StyledP>;
};

export default CopyrightFooter;
