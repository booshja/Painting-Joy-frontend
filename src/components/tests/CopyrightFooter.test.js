import { render } from "@testing-library/react";
import CopyrightFooter from "../CopyrightFooter";
import "jest-styled-components";

/** Smoke Test */
it("renders without crashing", () => {
    render(<CopyrightFooter />);
});

/** Snapshot Test */
it("matches snapshot", () => {
    const { asFragment } = render(<CopyrightFooter />);
    expect(asFragment()).toMatchSnapshot();
});

/** RTL Tests */
it("renders correctly", () => {
    const { getByText } = render(<CopyrightFooter />);

    expect(
        getByText("2021 - Painting Joy Mural Co.", { exact: false })
    ).toBeInTheDocument();
});
