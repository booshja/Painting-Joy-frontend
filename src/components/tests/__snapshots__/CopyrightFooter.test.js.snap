// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  .c0 {
  font-family: "Barlow Condensed",sans-serif;
  color: #ffffff;
  font-size: 1rem;
}

<p
    class="c0"
  >
    Copyright Â© 2021 - Painting Joy Mural Co.
  </p>
</DocumentFragment>
`;
